src/main/java/me/gnat008/perworldinventory/commands/SetWorldDefaultCommand.java

```diff
@@ -15,7 +15,7 @@
 public class SetWorldDefaultCommand implements ExecutableCommand {
 
     @Inject
-    private FileWriter fileSerializer;
+    private FlatFile fileSerializer;
```

Comment:
If you have operations that will always manipulate files (regardless of the settings) you should extract those methods into their separate class. It can get confusing fast otherwise as to why a concrete data source type is injected at some places. I'd wager that the methods used in this class are NOT part of the dataSource interface anyway.

Besides an issue of clarity it's important to be aware that if you inject a FlatFile directly, you'll get ANOTHER instance of FlatFile than if you inject DataSource (provided that it's mapped to FlatFile). This may bite you in the ass later on as it's hard to track down :P